src/dialogs/examples/basic/AdaptiveCardDialog.ts

```diff
@@ -0,0 +1,451 @@
+                                    "title": "I accept the terms and conditions (True/False)",
+                                    "valueOn": "true", // the value when toggle is on (default: true)
+                                    "valueOff": "false", // the value when toggle is off (default: false)
+                                    "id": "AcceptsTerms",
+                                    "isRequired": false,
+                                    "separation": null,
+                                },
+                            ],
+                        },
+                    ],
+                    "actions": [
+                        // submit action gathers up input fields, merges with optional data field and generates event to client asking for data to be submitted
+                        {
+                            "type": "Action.Submit",
+                            "title": "Submit",
+                            "speak": "<s>Search</s>",
+                            "data": {
+                                "dialog": "AdaptiveCardDialog",
+                            },
+                        },
+                        // show action defines an inline AdaptiveCard which is shown to the user when it is clicked
+                        {
+                            "type": "Action.ShowCard",
+                            "card": {
+                                "type": "AdaptiveCard",
+                                "version": "1.0",
+                                "body": [
+                                    {
+                                        "type": "Container",
+                                        "items": [
+                                            {
+                                                "type": "Input.Text",
+                                                "placeholder": "text here",
+                                                "style": "text",
+                                                "isMultiline": false,
+                                                "maxLength": 0,
+                                                "id": "Text",
+                                                "isRequired": false,
+                                                "speak": "<s>Please enter your text here?</s>",
+                                                "separation": null,
+                                            },
+                                        ],
+                                        "style": null,
+                                        "separation": null,
+                                    },
+                                ],
+                                "actions": [
+                                    {
+                                        "type": "Action.Submit",
+                                        "title": "Submit",
+                                        "speak": "<s>Search</s>",
+                                        "data": {
```

Comment:
Remove the 'dialog' property and add isFromAdaptiveCard: true. Next, add a 'messageText' property that is populated with a constant in this dialog - AdaptiveCardDialog.textToTriggerThisDialog = "Adaptive Card";

Reply:
Done.